src/esn.calendar.libs/app/components/time-select/time-select.controller.js

```diff
@@ -0,0 +1,59 @@
+'use stric';
+
+angular.module('esn.calendar.libs')
+  .controller('calTimeSelectController', calTimeSelectController);
+
+function calTimeSelectController(calMoment) {
+  const self = this;
+
+  self.$onInit = onInit;
+  self.selectedTimeChange = selectedTimeChange;
+  self.setSelectedTime = setSelectedTime;
+
+  function onInit() {
+    const initialTime = calMoment(this.date);
+
+    self.selectedTime = initialTime.format(self.timeFormat);
+    self.timeOptions = timeOptions();
+  }
+
+  function timeOptions() {
+    const options = [];
+
+    calMoment.locale(self.locale);
```

Comment:
This will change the locale globally I'm afraid.

You can change the locale locally by doing something like:

```
options.push(calMoment({ hour }).locale(self.locale).format(self.timeFormat));
```